tests/e2e/features/artifacts/depositPage/tokens.feature

```diff
@@ -0,0 +1,90 @@
+      | RAISE     | 0x3D79F1...46f | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/raise.svg?alt=media&token=79740de5-ac54-437b-a982-aec0a5b62361                                                                                                           |
+      | LSD       | 0x458A2E...bD4 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/lsd.svg?alt=media&token=8f20e143-4862-457e-9d7c-36da9676845f                                                                                                             |
+      | ETHx      | 0x668cc2...39c | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/ethx.svg?alt=media&token=362fe1c5-afb4-47cf-b766-c27aa11bde89&_gl=1*148cy0l*_ga*MTQ3NTc0NTE3Ni4xNjg2MDM4NzMy*_ga_CW55HF8NVT*MTY4NjYzODQ4Ny4xMy4xLjE2ODY2Mzg1NTguMC4wLjA. |
+      | WBTC      | 0xBBeB51...011 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/wbtc.svg?alt=media&token=502e8eee-d36a-4460-a97b-dad3e387cddd                                                                                                            |
+      | KNC       | 0x6ee46C...3e6 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/wbtc.svg?alt=media&token=502e8eee-d36a-4460-a97b-dad3e387cddd                                                                                                            |
+      | BEL       | 0xB83CFB...2D9 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/bel.png?alt=media&token=f8bbcfc2-6037-4626-870b-6a3b6630554b                                                                                                             |
+      | ZZ        | 0x1ab721...184 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/zz.png?alt=media&token=dc866379-92a9-4d85-9173-dad0076046b1                                                                                                              |
+
+  @id1564 @id1436 @tokens @mainnet
+  Scenario:  Check search functionality for Choose Tokens
+    Given I go to page "/transaction/zksync/era/deposit/?network=era-mainnet"
+    When I click by "testId" with "your-account" value
+    Then I click by "testId" with "token-dropDown" value
+    Then I fill the "data-testid=search_tokens" input field by "0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4"
+    Then Element with "text" "USDC" should be "visible"
+    Then I fill the "data-testid=search_tokens" input field by "0xd35CCeEAD182dcee0F148EbaC9447DAas2c4D449c4"
+    Then Element with "partial text" "Make sure you are using correct zkSync network" should be "visible"
+    Then Element with "xpath" "//*[@class='modal-card']//*[contains(., ' No tokens ')]" should be "visible"
+    Then Element with "xpath" "//*[@class='modal-card']//*[contains(., '0xd35CCeEAD182dcee0F148EbaC9447DAas2c4D449c4')]" should be "visible"
```

Comment:
Please, make this part more human readable: 
Element with "text" "No tokens was found" should be "visible"
Element with "text" "0xd35CCeEAD182dcee0F148EbaC9447DAas2c4D449c4" should be "visible"

Reply:
Also, please, add a new line #91